integration/tests/labels/label_test.go

```diff
@@ -0,0 +1,106 @@
+//go:build integration
```

Comment:
such a small feature, do we have to add a new suite :sweat_smile: ? Perhaps merge into CRUD (since it CR's a label :joy: ). Just to reduce the footprint

Reply:
Sure, I just thought that createdelete test is soo overloaded by now. But I can add this there too I guess. :)

Reply:
I guess I can set then set again. Should also be fine. The thing to test is that it can set multiple times anyways and that the previsouly set value is preserved. :)